README.md

```diff
@@ -397,6 +397,26 @@ for port 80.  If no port is available the load balancer will stay in Pending.
 To disable the embedded service load balancer (if you wish to use a different implementation like
 MetalLB) just add `--no-deploy=servicelb` to the server on startup.
 
+Air-Gap Support
+---------------
+
+k3s supports pre-loading of containerd images by placing them in the `images` directory for the agent before starting, eg:
+```sh
+sudo mkdir -p /var/lib/rancher/k3s/agent/images/
+sudo cp ./k3s-airgap-images-$ARCH.tar /var/lib/rancher/k3s/agent/images/
+```
+Images needed for a base install are provided through the releases page, additional images can be created with the `docker save` command.
+
+Offline Helm charts are served from the `/var/lib/rancher/k3s/server/static` directory, and HelmChart manifests may reference the static files with a `%{KUBERNETES_API}%` templated variable. For example, the default traefik manifest chart installs from `https://%{KUBERNETES_API}%/static/charts/traefik-X.Y.Z.tgz`.
```

Comment:
HelmChart should be one or two word ehre? `HelmChart manifests`

Reply:
They are manifests of kind HelmChart, split apart the word for better readability tho.